pmsoService/materials/services/material_order_services.py

```
@@ -186,4 +159,53 @@ def determine_required_storage_keepers(materials: list[Material]):
-    }
\ No newline at end of file
+    }
+
+
+def update_storage_import_status(material_order_id, request):
+    @transaction.atomic
+    def _update():
+        try:
+            instance = MaterialOrder.objects.get(id=material_order_id)
+        except MaterialOrder.DoesNotExist:
+            raise ValidationError(f"Material order with id {material_order_id} does not exist")
```

Ruff (0.14.3)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)